addOns/authhelper/src/main/zapHomeFiles/reports/auth-report-json/report.json

```diff
@@ -41,17 +41,20 @@
 			"created": [[${entry.createTimestamp}]],
 			"authenticationMethod": [[${entry.authenticationMethod}]],
 			"context": [[${entry.context}]],
-			"user": [[${entry.user}]]
+			"user": [[${entry.user}]],
```

Comment:
Add a test to generate and verify a JSON report? :)

Reply:
Will do.